src/Prettus/Repository/Eloquent/BaseRepository.php

```diff
@@ -622,7 +621,7 @@ public function create(array $attributes)
                 $attributes = $this->model->newInstance()->forceFill($attributes)->makeVisible($this->model->getHidden())->toArray();
             } else {
                 $model = $this->model->newInstance()->forceFill($attributes);
-                $model->addVisible($this->model->getHidden());
+                $model->makeVisible($this->model->getHidden());
```

Comment:
Is makeVisible available on L6.0 support ? or do you need to check laravel version and choose right one to use.

Reply:
Based on https://laravel.com/docs/5.7/eloquent-serialization#hiding-attributes-from-json (scroll down to Temporarily Modifying Attribute Visibility) makeVisible has been in Laravel since at least 5.1

Reply:
You can see https://github.com/laravel/framework/pull/31463 makeVisible was what was supposed to be used, addVisible should not have been and where undocumented.